### PR TITLE
Typos: xml/System.[M-S]*

### PR DESCRIPTION
**Pull request by nschonni**
Spell checking System.M* - System.S* files.
Will submit a PR to "samples" to adjust "Recieve" typo in examples

### CODE REVIEW

File: xml/System.Speech.Recognition/AudioSignalProblem.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The ccurredEventArgs.AudioSignalProblem property of the ccurredEventArgs class gets a member of this enumeration when the <xref:System.Speech.Recognition.SpeechRecognitionEngine> or <xref:System.Speech.Recognition.SpeechRecognizer> raises a <xref:System.Speech.Recognition.SpeechRecognitionEngine.AudioSignalProblemOccurred> event.  
+ The occurredEventArgs.AudioSignalProblem property of the occurredEventArgs class gets a member of this enumeration when the <xref:System.Speech.Recognition.SpeechRecognitionEngine> or <xref:System.Speech.Recognition.SpeechRecognizer> raises a <xref:System.Speech.Recognition.SpeechRecognitionEngine.AudioSignalProblemOccurred> event.
```
Comment (rpetrusha) [inline xml/System.Speech.Recognition/AudioSignalProblem.xml]:
```suggestion
 The <xref:System.Speech.Recognition.AudioSignalProblemOccurredEventArgs.AudioSignalProblem?displayProperty=nameWithType> property gets a member of this enumeration when the <xref:System.Speech.Recognition.SpeechRecognitionEngine> or <xref:System.Speech.Recognition.SpeechRecognizer> raises a <xref:System.Speech.Recognition.SpeechRecognitionEngine.AudioSignalProblemOccurred> event.  
```

File: xml/ns-System.Runtime.Remoting.Services.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- The <xref:System.Runtime.Remoting.Services.TrackingServices> class provides a tracking service with pluggable handlers that can detect when objects are marshaled and unmarshaled. Pluggable handlers can be created by implementing the <xref:System.Runtime.Remoting.Services.ITrackingHandler> interface. For more information, see the <xref:System.Runtime.InteropServices.Marshal> class.  
+ The <xref:System.Runtime.Remoting.Services.TrackingServices> class provides a tracking service with pluggable handlers that can detect when objects are marshaled and unmarshalled. Pluggable handlers can be created by implementing the <xref:System.Runtime.Remoting.Services.ITrackingHandler> interface. For more information, see the <xref:System.Runtime.InteropServices.Marshal> class.
```
Comment (rpetrusha) [inline xml/ns-System.Runtime.Remoting.Services.xml]: unmarshaled is the correct spelling.

File: xml/ns-System.Runtime.Remoting.xml [hunk 3]
```diff
@@ -9,7 +9,7 @@
...
   
- The <xref:System.Runtime.Remoting.ObjRef> class holds all the relevant information required to activate and communicate with a remote object. This class is a serializable representation of an object that is transmitted to a remote location using a channel, where it is unmarshaled (see <xref:System.Runtime.Remoting.RemotingServices.Unmarshal%2A>) and can be used to create a local proxy of the remoted object.  
+ The <xref:System.Runtime.Remoting.ObjRef> class holds all the relevant information required to activate and communicate with a remote object. This class is a serializable representation of an object that is transmitted to a remote location using a channel, where it is unmarshalled (see <xref:System.Runtime.Remoting.RemotingServices.Unmarshal%2A>) and can be used to create a local proxy of the remoted object.
```
Comment (rpetrusha) [inline xml/ns-System.Runtime.Remoting.xml]: unmarshaled is the correct spelling.

File: xml/System.Net.Sockets/Socket.xml [hunk 4]
```diff
@@ -6865,9 +6865,9 @@ The GetHashCode method returns a hash code of this instance. This value can be u
 ## Examples  
  The following receives the data found into `buffer`, and specifies <xref:System.Net.Sockets.SocketFlags.None> for <xref:System.Net.Sockets.SocketFlags>.  
   
- [!code-cpp[Socket_Send_Recieve#4](~/samples/snippets/cpp/VS_Snippets_Remoting/Socket_Send_Recieve/CPP/source.cpp#4)]
- [!code-csharp[Socket_Send_Recieve#4](~/samples/snippets/csharp/VS_Snippets_Remoting/Socket_Send_Recieve/CS/source.cs#4)]
- [!code-vb[Socket_Send_Recieve#4](~/samples/snippets/visualbasic/VS_Snippets_Remoting/Socket_Send_Recieve/VB/source.vb#4)]  
+ [!code-cpp[Socket_Send_Receive#4](~/samples/snippets/cpp/VS_Snippets_Remoting/Socket_Send_Receive/CPP/source.cpp#4)]
```
[the remark this answers is not in the document]
Comment (nschonni) [reply xml/System.Net.Sockets/Socket.xml]: Already merged https://github.com/dotnet/samples/pull/766

[the remark this answers is not in the document]
Comment (nschonni) [reply xml/System.Net.Sockets/Socket.xml]: Found an instance in dotnet/docs, but urelated to the files

**Comment by rpetrusha** [comment]
Thanks, @nschonni. We'll merge once the build completes successfully.

**Pull request merged**